### ISSUE TITLE
valid-jsdoc rule return preferred form is broken

### ISSUE THREAD

**Issue by Spy-Seth** [eslint/eslint#4238]
The `valid-jsdoc` still have issue (after https://github.com/eslint/eslint/issues/4205). It do not trigger error when we force the `@return` preferred form.

```javascript
/**
 * @return {Number}
 */
function foo() {
    return 1;
};

/**
 * @returns {Number}
 */
function bar() {
    return 2;
};

/**
 * No return tag.
 */
function bar() {
    return 2;
};
```

```json
{
  "rules": {
    "valid-jsdoc": [2, {
        "prefer": { "return": "return" }, 
        "requireParamDescription": false,
        "requireReturnDescription": false,
        "requireReturn": false
    }]
  }
}
```

Only one error is trigger with same config as above: 
```
15:1   error    Missing JSDoc @returns for function  valid-jsdoc
``` 

The `@returns` tag line 9 should also trigger an error.

**Comment by eslintbot** [comment]
Thanks for the issue! If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

**Comment by platinumazure** [comment]
From what I read of the documentation, I think the key of the `prefer`
object is the keyword you don't want, and the value is the one you do want.
Try `{ "returns": " return" }` maybe?

**Comment by Spy-Seth** [comment]
You're right! The documentation is not clear on this point!
